Integrations/MimecastV2/MimecastV2.py

```diff
@@ -443,43 +448,169 @@ def create_policy():
         'toValue': to_value
     }
 
+    return policy_obj, option
+
+
+def create_policy(args):
+    headers = ['Policy ID', 'Description', 'Sender', 'Reciever', 'Bidirectional', 'Start', 'End']
+    contents = {}  # type: Dict[Any, Any]
+    context = {}
+    policies_context = {}  # type: Dict[Any, Any]
```

Comment:
why is the initialisation needed?

Reply:
I can delete it, but I thought thats the way to do it cause it is the same in create_policy
Do you want me to delete it here and from create_policy ?